lib/msal-browser/docs/resources-and-scopes.md

```diff
@@ -0,0 +1,87 @@
+  
+The intended recipient of an **Access Token** is represented by the `aud` claim; in case the value for the `aud` claim does not mach the resource **APP ID URI**, the token should be considered invalid. Likewise, the permissions that an **Access Token** grants is represented by the `scp` claim. [See the documentation for further information](https://docs.microsoft.com/azure/active-directory/develop/access-tokens#payload-claims).
+
+## Working with Multiple Resources
+
+When you have to access multiple resources, initiate a separate token request for each:
+
+     ```javascript
+     // "User.Read" stands as shorthand for "graph.microsoft.com/User.Read"
+     const graphToken = await msalInstance.acquireTokenSilent({
+          scopes: [ "User.Read" ]
+     });
+
+     const customApiToken = await msalInstance.acquireTokenSilent({
+          scopes: [ "api://<myCustomApiClientId>/My.Scope" ]
+     });
+     ```
+
+> Bear in mind that you *can* request multiple scopes for the same resource (e.g. `User.Read`, `User.Write` and `Calendar.Read` for **MS Graph API**).
+
+In case you *erroneously* pass multiple resources in your token request, the token you will receive will only be issued for the first resource.
+
+     ```javascript
+     // you will only receive a token for MS GRAPH API's "User.Read" scope here
+     const myToken = await msalInstance.acquireTokenSilent({
+          scopes: [ "User.Read", "api://<myCustomApiClientId>/My.Scope" ]
+     });
+     ```
+
+## Dynamic Scopes and Incremental Consent
+
+In Azure AD, The scopes (*permissions*) set directly on the application registration are called **static**. Other scopes that are only defined within the code are called **dynamic**. This has implications on MSAL.js' **login** (i.e. *loginPopup*, *loginRedirect*) and **acquireToken** (i.e. *acquireTokenPopup*, *acquireTokenRedirect*, *acquireTokenSilent*) methods. Consider:
+
+     ```javascript
+          const loginRequest = {
+               scopes: [ "openid", "profile", "User.Read" ]
+          };
+
+          const tokenRequest = {
+               scopes: [ "Mail.Read" ]
+          };
+
+          // will return an ID Token and an Access Token with scopes: openid, profile, User.Read
+          msalInstance.loginPopup(loginRequest);
+
+          // will fail and fallback to an interactive method prompting a consent screen
+          msalInstance.acquireTokenSilent(tokenRequest);
+     ```
+
+In the code snippet above, the user will be prompted for consent once they authenticate and receive an **ID Token** and an **Access Token** with scope `User.Read`. Later, if they request an **Access Token** for `User.Read`, they will not be asked for consent again (in other words, they can acquire a token *silently*). On the other hand, the user did not consented to `Mail.Read` at the authentication stage. As such, they will be asked for consent when requesting an **Access Token** for that scope.
```

Comment:
> On the other hand, the user did not consented to `Mail.Read` at the authentication stage. As such, they will be asked for consent when requesting an **Access Token** for that scope.

Note that after consenting they will receive a new access token containing both `Mail.Read` and the previously consented `User.Read`, which is what we mean by "incremental consent"

Reply:
added!